app/src/main/java/org/mozilla/reference/browser/settings/AboutFragment.kt

```diff
@@ -43,30 +43,35 @@ class AboutFragment : Fragment() {
             val geckoVersion = PackageInfoCompat.getLongVersionCode(packageInfo).toString() + " \uD83E\uDD8E " +
                 MOZ_APP_VERSION + "-" + MOZ_APP_BUILDID
             String.format(
-                "%s (Build #%s)",
+                "%s (Build #%s)\n",
                 packageInfo.versionName,
                 geckoVersion
             )
         } catch (e: PackageManager.NameNotFoundException) {
             ""
         }
 
-        val versionInfo =
-            "\uD83D\uDCE6: ${Build.version}, ${Build.gitHash}\n\uD83D\uDEA2: ${Build.applicationServicesVersion}"
+        val versionInfo = String.format(
+            "%s \uD83D\uDCE6: %s, %s\n\uD83D\uDEA2: %s",
```

Comment:
We should port the gecko emoji code from [Fenix](https://github.com/mozilla-mobile/fenix/blob/30930cdc88508f7677f445ad6a2457c66d866aa7/app/src/main/java/org/mozilla/fenix/settings/AboutFragment.kt#L32) that falls back on pre-nougat devices.

```kt
val geckoEmoji = if (SDK_INT >= Build.VERSION_CODES.N) "\uD83D\uDEA2" else "GV"
```

Reply:
NEVERRRRRRR! 🦎 🦎 🦎 🦎 🦎 🦎 🦎 🦎

Reply:
We spend too much time talking about Geckos - let's remooooove it 😂

Reply:
Created [this issue](https://github.com/mozilla-mobile/reference-browser/issues/803) since we have multiple emojis on the About page now. 😬